policies/glossary.md

```diff
@@ -93,6 +104,14 @@ an ICLA.
 
 Also see: [CCLA](#ccla)
 
+## LDP
+
+_Linux Documentation Project_
+
+Our documentation is aims to broadly conform with [The Linux Documentation
+Projects] guidelines.
```

Comment:
should this be `[The Linux Documentation Project]'s` ?

Reply:
Also, the corresponding link reference `[The Linux Documentation
Project]: https://tldp.org` is missing at the end of the file.

Reply:
Good catches, thanks.